patroni/ctl.py

```diff
@@ -828,6 +828,80 @@ def members(obj, cluster_names, fmt, watch, w, extended, ts):
             cluster = dcs.get_cluster()
             output_members(cluster, cluster_name, extended, fmt)
 
+def output_topology(cluster, name):
```

Comment:
Why not to use existing function [print_output()](https://github.com/zalando/patroni/blob/6a505b837da98f275dfb10e5cf36a914f3a54e6b/patroni/ctl.py#L168)?
It would save from from quite a lot of ident calculations.

Reply:
These ident calculations are the source of this PR :) As they provide pretty and clear ASCII printing and view of cluster topology. I can move them to `print_output()` function and add some type of custom format in order to support this pretty printing. Will it be OK?

Reply:
Ok, will take a look on it and fix it. Thanks!

Reply:
done